src/timerUtils.ts

```diff
@@ -62,44 +67,69 @@ async function getTimers() {
   return setOfTimers;
 }
 
-async function renameTimer(timerFile: string, newName: string) {
+function renameTimer(timerFile: string, newName: string) {
   const dataPath = environment.supportPath + "/" + timerFile;
   writeFileSync(dataPath, newName);
 }
 
-async function ensureCTFileExists() {
+function ensureCTFileExists() {
   if (!existsSync(DATAPATH)) {
     writeFileSync(DATAPATH, JSON.stringify({}));
   }
 }
 
-async function createCustomTimer(newTimer: CustomTimer) {
-  await ensureCTFileExists();
+function createCustomTimer(newTimer: CustomTimer) {
+  ensureCTFileExists();
   const customTimers = JSON.parse(readFileSync(DATAPATH, "utf8"));
   customTimers[randomUUID()] = newTimer;
   writeFileSync(DATAPATH, JSON.stringify(customTimers));
 }
 
-async function readCustomTimers() {
-  await ensureCTFileExists();
+function readCustomTimers() {
+  ensureCTFileExists();
   const customTimers = JSON.parse(readFileSync(DATAPATH, "utf8"));
   return customTimers;
 }
 
-async function renameCustomTimer(ctID: string, newName: string) {
-  await ensureCTFileExists();
+function renameCustomTimer(ctID: string, newName: string) {
+  ensureCTFileExists();
   const customTimers = JSON.parse(readFileSync(DATAPATH, "utf8"));
   customTimers[ctID].name = newName;
   writeFileSync(DATAPATH, JSON.stringify(customTimers));
 }
 
-async function deleteCustomTimer(ctID: string) {
-  await ensureCTFileExists();
+function deleteCustomTimer(ctID: string) {
+  ensureCTFileExists();
   const customTimers = JSON.parse(readFileSync(DATAPATH, "utf8"));
   delete customTimers[ctID];
   writeFileSync(DATAPATH, JSON.stringify(customTimers));
 }
 
+function formatTime(seconds: number) {
```

Comment:
Out of curiosity... I understand extracting this out to the `timerUtils`, but why reimplement it?

Reply:
I had some issues where it would add an hour because of my timezone. 

However I just tried the old implementation and it works as expected now. Maybe I had something wrong in the beginning.
Should I keep the current implementation or change it to yours?

I prefer yours as it is way shorter.